docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,151 @@
+
+# Widget Control
+
+The widget control provides information about installing and running widget applications.
+
+It also provides functions for the following:
+
+*   Sending update requests to the widget applications
+*   Retrieving details of running instance for the same package widget applications
+
+
+The main features of the `Tizen.Applications.WidgetControl` class include:
+
+-   [Getting information on widget applications](#getting_information)
+
+    For installed (but not necessarily running) widget applications, you can retrieve widget information with the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class.
+
+
+-   [Listening widget lifecycle events on widget applications](#listening_events)
+
+    For running widget applications, you can listen widget application lifecycle events with the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class.
+
+
+-   [Communicating with running widget instances](#communicating_instances)
+
+    For running widget instances, you can trigger an update event, send update data, and retrieve running widget instances detail with the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class.
+
+
+## Prerequisites
+
+To enable your application to use the widget control functionality:
+
+1.  To use the methods and properties of the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class, include the [Tizen.Applications](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.html) namespace in your application:
+
+    ```
+    using Tizen.Applications;
+    ```
+
+2.  To get information on widget application, the application has to request permission by adding the following privilege to the  `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/widget.viewer</privilege>
+    </privileges>
+    ```
+
+<a name="getting_information"></a>
+## Getting Information on Widget Applications
+
+To get information on the widget applications, follow these steps:
+
+1.  Create an instance of the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class with the ID of the widget application, as shown in the following code:
+
+    ```
+    WidgetControl control = new WidgetControl(Your Widget ID);
+    ```
+
+2.  Operate on the control:
+    -   Get the main application ID, package ID, and available scale lists from the control:
+
+        ```
+        string mainId = control.MainAppId;
+        string packageId = control.PackageId;
+        IEnumerable<WidgetControl.Scale> scales = control.GetScales();
+        ```
+
+
```

Comment:
Please remove extra space from line 66 and 67.

Reply:
Done